app/src/main/java/net/squanchy/speaker/view/SpeakerViewHolder.java

```diff
@@ -27,7 +27,7 @@ public void updateWith(Speaker speaker, @Nullable SpeakersView.OnSpeakerClickedL
         speakerImage.setImageResource(R.mipmap.ic_launcher); //TODO load the picture from the url provided in the speaker object
         itemView.setOnClickListener(v -> {
             if (listener != null) {
-                listener.onSpeakerClicked(speaker.id());
+                listener.onSpeakerClicked(speaker.numericId());
```

Comment:
❌  Why? This should either be the "real" ID we can use to query, or the whole `speaker` (which, probably makes more sense here)